### PR TITLE
some more child death scenarios

### PR DESCRIPTION
**Pull request by maxx0911**
Children dying in the bot is one of the saddest and yet funniest things you can see, and because of that, i decided to add some more causes of death as seen in channel #feedback on the support server. i couldn't get all of them into that response, so that it would make sense grammarly though.

### CODE REVIEW

**Comment by Gelbpunkt** [comment]
This is grammatically not always correct. I would suggest making the message more dynamic.
`Child died at the age of X! {Text goes here}`

That way every cause can be added easily.

Can you change it to that? Also, Catholic church is legit to me :)

**Comment by maxx0911** [comment]
Alright, done. Maybe a gender thing would be cool too? Instead of always saying "they" to keep it gender-neutral, it could also be "he" or "she", depending on the gender of the child.

**Comment by Gelbpunkt** [comment]
Final thing: Maybe not always the `They xy` pattern, try `They've been abducted by aliens!` e.g.

Is the `\` before the © needed?

**Comment by maxx0911** [comment]
I'm not the best english speaker out there, but I'll get some changes in too.
And no, the `\` isn't necessary. It would just look better to display © as the text caharcter, rtther than the emoji, because on the dark gray Discord background, a black emoji is kind of hard to see.

**Comment by Gelbpunkt** [comment]
Agreed, we should escape it then

**Comment by maxx0911** [comment]
There we go, should now be ready to merge :)

**Comment by Gelbpunkt** [comment]
https://github.com/Gelbpunkt/IdleRPG/blob/v3.4-dev/cogs/marriage.py#L374
Don't forget to change this line

**Comment by maxx0911** [comment]
sorry if this a dumb questio, but what should be changed there? It's like you suggested.
`f"{target[2]} died at the age of {target[3]}! {cause}."` where `target[2]` is the child's name, `target[3]` is the age and `cause` is the sentence that'll be sent.

**Comment by Gelbpunkt** [comment]
Exactly

**Comment by maxx0911** [comment]
as you can see here, I have already changed that :eyes: 
https://github.com/Gelbpunkt/IdleRPG/blob/819a0e9c985ff4290fdc5e3987949523cdf28926/cogs/marriage.py#L383

**Comment by Gelbpunkt** [comment]
Did you run black? If yes, comment and I'll merge

**Comment by maxx0911** [comment]
Yea, go ahead.

File: cogs/marriage.py [hunk 1]
```diff
@@ -357,7 +357,21 @@ def check(msg):
...
+                "They died of lovesickness", 
+                "They've died of age", 
+                "They died of loneliness", 
+                "A horde of goblins got them.", 
+                "They have finally decided to move out after all these years, but couldn't survive a second alone", 
+                "Spontanious combustion removed them from existence",
+                "While exploring the forest, they have gotten lost",
+                "They've left through a portal into another dimension",
+                "The unbearable pain of stepping on a Lego\© brick killed them",
+                "You heard a landmine going off nearby..",
```
Comment (numbermaniac) [inline cogs/marriage.py]: There's two full stops at the end of this line.

File: cogs/marriage.py [hunk 2]
```diff
@@ -357,7 +357,21 @@ def check(msg):
...
+                "They have been abducted by aliens",
+                "The Catholic Church got them..",
```
Comment (numbermaniac) [inline cogs/marriage.py]: Here as well.

**Comment by numbermaniac** [comment]
If those are supposed to be "trailing off" sentence then they should have 3 dots rather than 2.

**Comment by maxx0911** [comment]
The third dot is always there, after `{cause}` for the trailing off ones.

**Comment by numbermaniac** [comment]
Yeah...you're right, I just realised. My mistake.

**Comment by Gelbpunkt** [comment]
@numbermaniac @maxx0911 my commit fixed all of that haha

**Pull request merged**